build/pipelines/templates-v2/steps-install-vcpkg.yml

```diff
@@ -0,0 +1,13 @@
+steps:
+  - pwsh: |-
+      $VsInstallRoot = & 'C:\Program Files (x86)\Microsoft Visual Studio\Installer\vswhere.exe' -requires Microsoft.VisualStudio.Component.Vcpkg -property installationPath
+      $VcpkgRoot = Join-Path $VsInstallRoot "VC\vcpkg"
+      If ([String]::IsNullOrEmpty($VcpkgRoot)) {
+        Remove-Item -Recurse -Force dep/vcpkg -ErrorAction:Ignore
+        git clone https://github.com/microsoft/vcpkg dep/vcpkg

```

Comment:
This needs the full history for version pinning right? (Otherwise, we could do a shallow clone.)

Reply:
I believe so, yes.